src/Guest/Guest.js

```diff
@@ -2,12 +2,64 @@ import React, { Component } from "react";
 import "./Guest.scss";
 
 class Guest extends Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = {
-      
+// do we need it to be a class?
```

Comment:
Is there ever going to be a state change with guest? If it was a generic User component I would say definitely say yes. Does guest need to even be a component? Or should it just be a property passed to different Components?

Reply:
It depends on the direction we want to go with this project if we decide to use hooks we wouldn't need it to be a class component.